src/sentry/tsdb/redis.py

```diff
@@ -727,7 +726,9 @@ def get_most_frequent(
         results = {}
         cluster, _ = self.get_cluster(environment_id)
         for key, responses in cluster.execute_commands(commands).items():
-            results[key] = [(member, float(score)) for member, score in responses[0].value]
+            results[key] = [
+                (member.decode("utf-8"), float(score)) for member, score in responses[0].value
+            ]
```

Comment:
I think this and `_series` are actually unused and can be removed, I ran across this when looking at TSDB before.